submission/distributor.go

```diff
@@ -39,13 +39,15 @@ var (
 	distOnce      sync.Once
 	reqsCounter   monitoring.Counter   // logurl, ep => value
 	rspsCounter   monitoring.Counter   // logurl, ep, sc => value
+	errCounter    monitoring.Counter   // logurl, ep, status => value
 	logRspLatency monitoring.Histogram // logurl, ep => value
 )
 
 // distInitMetrics initializes all the exported metrics.
 func distInitMetrics(mf monitoring.MetricFactory) {
 	reqsCounter = mf.NewCounter("http_reqs", "Number of requests", "logurl", "ep")
 	rspsCounter = mf.NewCounter("http_rsps", "Number of responses", "logurl", "ep", "httpstatus")
+	errCounter = mf.NewCounter("err_count", "Number of errors", "logurl", "ep", "statuscode")
```

Comment:
I'd rename `statuscode` to something more generic, like `errortype` / `errtype` or just `error`. Usually "status code" is associated with a more or less "standard" integer code or its name, whereas this metric is custom.

Reply:
Done